apps/server/src/setup/index.ts

```diff
@@ -109,13 +109,9 @@ export const publicDir = {
   /** path to uploads folder */
   uploadsDir: join(resolvePublicDirectory, config.uploads),
   /** path to external folder */
-  externalDir: externalsStartDirectory,
+  externalDir: join(resolvePublicDirectory, config.external),
```

Comment:
I am assuming that the fix is related to this somehow,

However, I think we still want to use `externalsStartDirectory`
In development mode, that provides us the folder at src, which makes it easier for us to work with

Maybe I am missing something?